libs/feature/record/src/lib/data-view-table/data-view-table.component.ts

```diff
@@ -111,7 +113,7 @@ export class DataViewTableComponent {
     } else if (this.linkHelper.isEsriRestFeatureServer(link)) {
       const url = getEsriRestDataUrl(link, 'geojson')
       return fromPromise(
-        readDataset(url, 'geojson').then((features) =>
+        readDataset(this.proxy.getProxiedUrl(url), 'geojson').then((features) =>
```

Comment:
again, i would have done a shared `readDatasetFn` somewhere, it's like for fetching the dataset we have to change the code at 3 or 4 different places.

Reply:
agreed!